docs/develop/configure-your-add-in-to-use-a-shared-runtime.md

```diff
@@ -49,7 +49,10 @@ Follow these steps for a new or existing project to configure it to use a shared
     <DefaultSettings>
     ```
 
-1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section just inside the `<Host ...>` tag. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
+1. Find the `<VersionOverrides>` section and add the following `<Runtimes>` section. The lifetime needs to be **long** so that your add-in code can run even when the task pane is closed. The `resid` value is **Taskpane.Url**, which references the **taskpane.html** file location specified in the ` <bt:Urls>` section near the bottom of the **manifest.xml** file.
+
+> [!IMPORTANT]
+> The `<Runtimes>` section must be entered after the `<Host>` element in the exact order as shown in the following XML.
 
    ```xml
```

Comment:
Indent the entire xml sample with 4 spaces else it breaks the numbering for this section

Reply:
thanks, fixed!